browser/brave_content_browser_client.cc

```diff
@@ -88,6 +89,11 @@ using extensions::ChromeContentBrowserClientExtensionsPart;
 #include "brave/components/ipfs/ipfs_navigation_throttle.h"
 #endif
 
+#if BUILDFLAG(UNSTOPPABLE_DOMAINS_ENABLED)
+#include "brave/browser/unstoppable_domains/unstoppable_domains_service_factory.h"
```

Comment:
is it needed?

Reply:
It's a left-over, will remove, thanks.

Reply:
Fixed in https://github.com/brave/brave-core/pull/7702/commits/cf30603be56f77af3d8bd829007e66246640293f